src/ci/github-actions/ci.yml

```diff
@@ -675,8 +675,9 @@ jobs:
                 --host=aarch64-pc-windows-msvc
                 --enable-full-tools
                 --enable-profiler
-              SCRIPT: python x.py dist
-              DIST_REQUIRE_ALL_TOOLS: 1
+              SCRIPT: python x.py dist bootstrap --include-default-paths
+              # RLS does not build for aarch64-pc-windows-msvc. See rust-lang/rls#1693
+              DIST_REQUIRE_ALL_TOOLS: 0
```

Comment:
this looks unintentional

Reply:
do i need to remove bootstrap from this workflow step ?